com.unity.visualeffectgraph/Editor/GraphView/Views/VFXView.cs

```diff
@@ -598,45 +571,35 @@ private void OnOpenAttachMenu()
             attachPanel.ShowAsDropDown(bounds, attachPanel.WindowSize, new[] { PopupLocation.BelowAlignLeft });
         }
 
-        void OnRefreshUI(DropdownMenuAction action)
-        {
-            Resync();
-        }
-
-        void OnRuntimeModeChanged(DropdownMenuAction action)
+        internal void ToggleRuntimeMode()
         {
             m_IsRuntimeMode = !m_IsRuntimeMode;
             controller.graph.SetCompilationMode(m_IsRuntimeMode ? VFXCompilationMode.Runtime : VFXCompilationMode.Edition);
         }
 
-        DropdownMenuAction.Status RuntimeModeStatus(DropdownMenuAction action)
-        {
-            if (m_IsRuntimeMode)
-                return DropdownMenuAction.Status.Checked;
-            else
-                return DropdownMenuAction.Status.Normal;
-        }
+        internal bool GetShaderValidation() => m_ForceShaderValidation;
 
-        void OnShaderValidationChanged(DropdownMenuAction action)
+        internal void ToggleShaderValidationChanged()
         {
             m_ForceShaderValidation = !m_ForceShaderValidation;
             controller.graph.SetForceShaderValidation(m_ForceShaderValidation);
         }
 
-        DropdownMenuAction.Status ShaderValidationStatus(DropdownMenuAction action)
-        {
-            if (m_ForceShaderValidation)
-                return DropdownMenuAction.Status.Checked;
-            else
-                return DropdownMenuAction.Status.Normal;
-        }
-
         [NonSerialized]
         Dictionary<VFXModel, List<IconBadge>> m_InvalidateBadges = new Dictionary<VFXModel, List<IconBadge>>();
 
         [NonSerialized]
         List<IconBadge> m_CompileBadges = new List<IconBadge>();
 
+        private void SetToolbarEnabled(bool enabled)
```

Comment:
Even when no asset is opened we want the help button to stay available, so we don't disable the whole toolbar anymore